specs/workflows/steps/index.spec.js

```diff
@@ -3947,6 +3947,27 @@ describe("shared workflow steps", () => {
 			});
 		});
 
+		it("should resolve if on develop branch", () => {
```

Comment:
Maybe a bit more specific language like "should resolve and not prompt if on develop branch"? (same for others)